pages/blog/_slug.vue

```diff
@@ -292,14 +292,11 @@ export default {
       },
       headline: this.article.title,
       description: this.article.description,
-      metaTitle: this.article.metaTitle,
-      metaDescription: this.article.metaDescription,
```

Comment:
Don't we need `metaTitle` and `metaDescription` for schema ?

Reply:
Yes, we validated this through validator.schema.org, metaTitle and metaDescription are not required in schema. 
<img width="694" alt="Screenshot 2023-05-25 at 12 41 18 PM" src="https://github.com/formester/formester-site/assets/132327557/e5057962-e1b9-465a-a2f7-1467caf014d2">

Reply:
okay